### PR TITLE
feat: show real values for current network balance

### PR DESCRIPTION
**Pull request by lynnshaoyu**
show current user balance in token detail for current network. show nothing when balance is zero. 

Screen Shot 2022-09-02 at 3 17 20 PM

### CODE REVIEW

**Comment by callil** [comment]
We should hide the card if the user has 0 balance.

**Comment by lynnshaoyu** [comment]
> 

cool thanks for calling out, i made the fix so now it shows nothing when balance is zero

File: src/components/Tokens/TokenDetails/BalanceSummary.tsx [hunk 1]
```diff
@@ -33,14 +33,14 @@ const ErrorText = styled.span`
...
-`
+// const NetworkBalancesSection = styled.div`
+//   height: fit-content;
+// `
 const TotalBalanceSection = styled.div`
   height: fit-content;
-  border-bottom: 1px solid ${({ theme }) => theme.backgroundOutline};
-  margin-bottom: 20px;
-  padding-bottom: 20px;
+  // border-bottom: 1px solid ${({ theme }) => theme.backgroundOutline};
```
[the remark this answers is not in the document]
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/BalanceSummary.tsx:39]: sg done

File: src/components/Tokens/TokenDetails/BalanceSummary.tsx [hunk 2]
```diff
@@ -63,51 +63,67 @@ export default function BalanceSummary({
...
-  const tokenSymbol = useToken(address)?.symbol
-  const { loading, error, data } = useNetworkTokenBalances({ address })
+  const token = useToken(address)
+  const { loading, error } = useNetworkTokenBalances({ address })
 
-  const { chainId: connectedChainId } = useWeb3React()
+  const { account } = useWeb3React()
+  const balance = useTokenBalance(account, token ?? undefined)
+  const balanceNumber = balance ? formatToDecimal(balance, Math.min(balance.currency.decimals, 6)) : undefined
+  const balanceUsd = useStablecoinValue(balance)?.toFixed(2)
```
[the remark this answers is not in the document]
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/BalanceSummary.tsx:59]: can't, as discussed in slack

**Review by JackShort — APPROVED** [review]

**Pull request merged**